recipes/reduct-cpp/all/conanfile.py

```diff
@@ -0,0 +1,107 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv
+import os
+
+required_conan_version = ">=1.53.0"
+
+class ReductCPPConan(ConanFile):
+    name = "reduct-cpp"
+    description = "ReductStore Client SDK for C++"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/reductstore/reduct-cpp"
+    topics = ("reductstore", "sdk", "http-client")
+    package_type = "static-library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "with_std_chrono": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "with_std_chrono": False,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return 20
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "11.2",
+            # spaceship operator(<=>) in std lib has been supported since clang 14
+            "clang": "14",
+            # spaceship operator(<=>) in std lib has been supported since apple-clang 14
+            "apple-clang": "14",
+            "Visual Studio": "16",
+            "msvc": "192",
+        }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("openssl/[>=1.1 <4]")
+        self.requires("fmt/11.0.2")
+        self.requires("cpp-httplib/0.18.0")
+        self.requires("nlohmann_json/3.11.3")
+        self.requires("concurrentqueue/1.0.4")
+        self.requires("date/3.0.1")
```

Comment:
```suggestion
        if not self.options.with_std_chrono:
            self.requires("date/3.0.1")
```

If I understand correctly, date is only required when not using std::chrono 

https://github.com/reductstore/reduct-cpp/blob/main/cmake/InstallDependencies.cmake#L56